integration

```diff
@@ -660,6 +660,7 @@
   "nagyrobi/home-assistant-custom-components-linkplay",
   "nathanmarlor/foxess_modbus",
   "Nazze/ha_best_bottrop_garbage_collection",
+  "nberktumer/ha-bambu-lab-p1-spaghetti-detection",
```

Comment:
This is an add-on repository.
Why does this PR even exist? The add-on can write the companion integration and blueprint.
Look at how the HACS-get add-on work.